classes/converter.php

```diff
@@ -17,6 +17,10 @@
 namespace tool_pdfpages;
 
 use moodle_url;
+use setasign\Fpdi\TcpdfFpdi;
```

Comment:
What's this `setasign` thing? Could we use `pdf` from **lib/pdflib.php** instead?

Reply:
Thanks for discussing this one with me @tsmilan. I always thought `assignfeedback_editpdf` was using core's PDF wrapper to do the stitching, but after checking with you, I see it bundles this `TcpdfFpdi` thing in the feedback plugin itself and uses that to join pages

It would be nice if there was something in core we could use to join the pages, but as the `TcpdfFpdi` thing from assign is part of the feedback plugin, it doesn't seem appropriate to use that here. So I think what you've done in bundling it in to this plugin itself is appropriate.